packages/preview/examination-paper/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "examination-paper"
```

Comment:
I think the name is a bit too descriptive given what your template aims at. Can you please change it to be a bit more creative and unique?